### PR TITLE
Add bottom navigation

### PR DESCRIPTION
**Pull request by Mrtenz**
This adds a bottom navigation menu with:

- A link to the GitHub page
- A button to open the settings modal

### CODE REVIEW

File: src/features/configuration/Configuration.tsx [hunk 1]
```diff
@@ -48,61 +52,50 @@ export const Configuration = () => {
...
-              <FormLabel>Local server location</FormLabel>
-              <Input
-                type="text"
-                value={snapUrl}
-                onChange={handleSnapUrlChange}
-              />
+    <Modal
+      isOpen={isOpen}
+      onClose={handleClose}
+      closeOnOverlayClick={true}
```
Comment (FrederikBolding) [inline src/features/configuration/Configuration.tsx]:
```suggestion
```
This is the default IIRC

File: src/features/configuration/Configuration.tsx [hunk 2]
```diff
@@ -48,61 +52,50 @@ export const Configuration = () => {
...
+      closeOnEsc={false}
```
Comment (FrederikBolding) [inline src/features/configuration/Configuration.tsx]:
```suggestion
```

File: src/features/configuration/Configuration.tsx [hunk 3]
```diff
@@ -48,61 +52,50 @@ export const Configuration = () => {
...
-                />
-              </HStack>
-            </FormControl>
-          </ModalBody>
+            <HStack mt="4" alignItems="center" justifyContent="space-between">
+              <FormLabel mb="0" htmlFor="ses-switch">
+                Secure EcmaScript (SES)
+              </FormLabel>
+              <Switch
+                id="ses-switch"
```
Comment (FrederikBolding) [inline src/features/configuration/Configuration.tsx:82]: Readonly for now?

File: src/features/configuration/Configuration.tsx [hunk 4]
```diff
@@ -48,61 +52,50 @@ export const Configuration = () => {
...
+        <Divider my="4" />
+        <ModalBody pt="0">
+          <FormControl>
+            <FormLabel>Local server location</FormLabel>
+            <Input type="text" value={snapUrl} onChange={handleSnapUrlChange} />
 
-              <FormLabel mt="4">Environment SRP</FormLabel>
-              <Textarea value={srp} onChange={handleSrpChange} />
+            <FormLabel mt="4">Environment SRP</FormLabel>
+            <Textarea value={srp} onChange={handleSrpChange} />
```
Comment (FrederikBolding) [inline src/features/configuration/Configuration.tsx]: Readonly for now?

File: src/features/status/StatusIndicator.tsx [hunk 5]
```diff
@@ -17,8 +18,12 @@ export const StatusIndicator = () => {
       ? 'text.error'
       : 'info.default';
 
+  const handleClick = () => {
+    dispatch(openConfigurationModal());
+  };
+
   return (
-    <>
+    <Flex alignItems="center" gap="2" onClick={handleClick} cursor="pointer">
```
Comment (FrederikBolding) [inline src/features/status/StatusIndicator.tsx]: Can we add a hover state?

Comment (Mrtenz) [reply src/features/status/StatusIndicator.tsx]: I think `cursor: pointer` is enough?

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**